episodes/designing-a-light-microscopy-experiment.md

```diff
@@ -0,0 +1,402 @@
+
+There are many different types of confocal microscopes, but one of the most 
+common is: laser scanning confocal microscopes. These are also known as point 
+scanning confocal microscopes. In this microscope, a laser is focused onto a 
+single point of the specimen at a time (rather than illuminating the entire 
+sample). The point of illumination is then moved across the sample in a raster 
+pattern (i.e. line by line), with light detected point by point. This gradually 
+builds up the final image.
+
+Confocal microscopes are typically used for fluorescence microscopy, especially 
+when it is necessary to image the shape of structures in full 3D. Bear in mind 
+that their advantages don't come without other disadvantages! For example, 
+confocal microscopes will generally be more complex to use and slower to acquire 
+images than a widefield microscope. Although there are other types of confocal 
+that are faster than laser scanning systems, such as 'spinning disc confocal 
+microscopes'.
+
+![The image above is Napari's Kidney (3D + 3Ch) sample image. This was acquired 
+with confocal fluorescence microscopy.](fig/confocal-example.png){alt="Screenshot 
+of Napari's Kidney (3D + 3Ch) sample image" width='60%'}
+
+### Super-resolution
+
+As there are so many [different 'super-resolution' methods
+](https://royalsocietypublishing.org/doi/10.1098/rsta.2021.0110) available, we 
+will only give a brief mention here. The key point is that super-resolution 
+methods provide extremely high resolution images that go beyond the classic 
+limit set by the diffraction of light. Many of these systems are based on 
+modified widefield or confocal microscope setups. This increase in resolution 
+usually comes at the cost of increased complexity in experiment/microscope setup.
+
+### Choosing a method
+
+Choosing a light microscopy method is about finding the simplest and fastest 
+approach that can address your research question. For example, does it provide 
+the resolution you need? Can it provide good images of thick vs thin samples? 
+Does it image a large enough area? Does it image fast enough to provide the 
+temporal resolution you need?... The online [bioimaging guide
+](https://www.bioimagingguide.org/welcome.html) provides a [useful flowchart
+](https://www.bioimagingguide.org/02_Sample_acquisition/Picking.html) to give 
+you some ideas - as well as links to many other great microscopy resources.
+
+If your research question can be solved by a simpler and less costly method, 
+then usually this is the way to go! It's all about choosing the right tool for 
+the job - different approaches will be best for different research questions. 
+For example, you could count cell number with a high-end super-resolution 
+microscope, but this is also possible with a standard widefield which will be 
+simpler and faster. In general, only move to more complex techniques when your 
+research question really requires it.
```

Comment:
Sounds like good advice for statistical models and image processing algorithms as well.